README.md

```diff
@@ -19,29 +19,56 @@ Harness Feature Flags (FF) is a feature management solution that enables users t
 ![FeatureFlags](https://github.com/harness/ff-php-server-sdk/raw/main/docs/images/ff-gui.png)
 
 ## Requirements
-
-
+To use this SDK, make sure you’ve:
+- Installed [PHP](https://www.php.net/) 7.4 or a newer version
+- [Composer](https://getcomposer.org/) dependency Manager for PHP
 ### General Dependencies
-
-
+- [Relay Proxy](https://github.com/harness/ff-proxy)
+- [Redis](https://redis.io/)
 ## Quickstart
 The Feature Flag SDK provides a client that connects to the feature flag service, and fetches the value
 of feature flags. The following section provides an example of how to install the SDK and initialize it from an application.
 
 This quickstart assumes you have followed the instructions to [setup a Feature Flag project and have created a flag called `harnessappdemodarkmode` and created a server API Key](https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag#step_1_create_a_project).
 
-### Install the FF SDK Dependency
+### Install the SDK Dependency
 
-The first step is to install the FF SDK as a dependency in your application using your application's dependency manager. 
+The first step is to install the SDK as a dependency in your application using your application's dependency manager.
 
+```
+composer require harness/ff-server-sdk
+```
 ### A Simple Example
 
 After installing the SDK, enter the SDK keys that you created for your environment. The SDK keys authorize your application to connect to the FF client. 
 
 ```php
-<?php echo "simple example"; ?>
+<?php
+
+require_once realpath("vendor/autoload.php");
+
+use Harness\CFClient;
+use OpenAPI\Client\Model\Target;
+
+$FLAG_KEY = "harnessappdemodarkmode";
+$SDK_KEY = getenv("SDK_KEY") ?: "";  // you can put your key in env variable or you can provide in the code
+
+$client = new CFClient($SDK_KEY, new Target(["name" => "Harness", "identifier" => "harness"]), [
+	"base_url": "http://proxy-url",
+	"events_url": "http://proxy-url",
+]);
+
+echo "Evaluation value for flag $FLAG_KEY with target 'enver': " . $client->evaluate($FLAG_KEY, false);
+$client->close();
 ```
 
+## Running the example
+
+To run the example, open the browser and type:
+
+```
+http://localhost/yourscriptfile.php
```

Comment:
Need to replace these instructions on how to set up and run the example... either offline or online example and perhaps with the docker command.

Reply:
Done